### DISCUSSION TITLE
1.5

### DISCUSSION THREAD

**Discussion by kirb** [LegacyUpdate/LegacyUpdate#87] [Announcements]
Yesterday, January 10, 2023, was the [end of support for Windows 8.1](https://support.microsoft.com/en-us/windows/windows-8-1-support-will-end-on-january-10-2023-3cfd4cde-f611-496a-8057-923fba401e93). Legacy Update 1.5 adds support for getting Windows 8 and 8.1 up to date. As a side effect, Windows Server 2012 and 2012 R2 are also supported starting with this release, despite that these remain officially supported by Microsoft [until October](https://learn.microsoft.com/en-us/lifecycle/announcements/windows-server-2012-r2-end-of-support).

#### Other changes in this release:

* While updating root certificates, Legacy Update will now also enable the TLS v1.1 and v1.2 protocols, which are present but disabled by default in Windows 7 and 8, as well as in Windows XP with [a POSReady 2009 update](https://support.microsoft.com/kb/4019276). This is an additional fix for websites not loading in Internet Explorer and other apps that use the Windows built-in networking functionality.
* Fixes crash on launch when opening Legacy Update setup on Windows XP SP3.
* Fixes Legacy Update setup not detecting the prerequisite updates required for Windows Server 2008 and 2008 R2.
* Fixes the Legacy Update shortcut incorrectly launching 64-bit Internet Explorer rather than 32-bit in some cases on Windows XP x64 Edition. Thanks to @stdin82 for helping to point out the fix for this.

<hr /><em>This discussion was created from the release <a href='https://github.com/kirb/LegacyUpdate/releases/tag/v1.5.0.0'>1.5</a>.</em>

**Comment by Jaguarek62** [comment]
tls 1.1 and 1.2 are enabled by default in windows 8.1

**Reply by kirb** [reply]
You’re right, thanks. That should have just said 8, not both 8 and 8.1.
